### PR TITLE
[MAINT] - Update contributing guidelines, author links, and lesson outline

### PR DESCRIPTION
**Pull request by jerdra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,40 @@
...
+
+This lesson covers fMRI imaging analysis from the basic steps of preprocessing and data cleaning, to running an analysis to explore connectivity patterns in the brain.
+
+### Episodes
+
+| Time | Episode | Question(s) Answered |
+| ---  | ---     | ---                  |
+||Setup|Download files required for the lesson|
+| 00:00 | 1. Exploring Preprocessed fMRI Data from fMRIPREP   | How does fMRIPrep store preprocessed neuroimaging data. How do I access preprocessed neuroimaging data                                     |
+| 00:25 | 2. Introduction to Image Manipulation using Nilearn | How can be perform arithmetic operations on MR images                                                                                                  |
```
Comment (kaitj) [inline README.md:30]: From "How can **be** perform arithmetic operations on MR images" to "How can **we** perform arithmetic operations on MR images"

File: README.md [hunk 2]
```diff
@@ -4,6 +4,40 @@
...
+| 01:55 | 4. Preprocessing fMRI Data                          | What are the standard preprocessing steps? What existing pipelines help with preprocessing?                                                                                                              |
+| 02:25 | 5. Cleaning Confounders in your Data with Nilearn   | How can we clean the data so that it more closely reflects BOLD instead of artifacts                                                                                                                                         |
+| 03:05 | 6. Applying Parcellations to Resting State Data     | How can we reduce amount of noise-related variance in our data? How can we frame our data as a set of meaningful features?                                                                                                 |
+|03:50|7. Functional Connectivity Analysis|How can we estimate brain functional connectivity patterns from resting state data?|
```
Comment (kaitj) [inline README.md:35]: From lines 35 - 39, just spacing between the columns (can't remember if this affects how it looks visually).

File: README.md [hunk 3]
```diff
@@ -4,6 +4,40 @@
 [![Slack Status](https://img.shields.io/badge/Slack_Channel-neuroimaging-E01563.svg)](https://swcarpentry.slack.com/messages/CCJBHKCHZ)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/carpentries-incubator/SDC-BIDS-fMRI/gh-pages)
 
+## Background
+
+This is one sub-module within the [Neuroimaging cirriculum][neuro_cirriculum]. Visit the link to view all the modules associated with the Neuroimaging Carpentries program.
+
+**fMRI Analysis in Python** is a programme developed to faciliate reproducible analysis in the area of functional neuroimaging analysis. Python is emerging as a standard language of data analysis, visualization, and workflow building. More recently, it has rapidly been adopted by the neuroimaging community as a means of developing powerful open-source tools in favour of historially used opaque packages such as AFNI, FSL and SPM. In addition, the barrier to entry to python is low - meaning that you as the user can easily develop your own packages and contribute to the open-source codebase of neuroimaging!
```
Comment (kaitj) [inline README.md:11]: "...programme developed to **faciliate**..." to "...programme developed to **facilitate**..."

**Review by kaitj — CHANGES_REQUESTED** [review]
Overall looks good to me - just a few spelling changes.

Can't recall the markdown format exactly for tables either, so the spacing thing mentioned in the review comments may not even be an issue.

File: README.md (hunk 2 above)
Comment (jerdra) [reply README.md:35]: haha sorry that's my editor's doing - it renders fine though!

File: README.md (hunk 1 above)
Comment (jerdra) [reply README.md:30]: thanks for the catch!

File: README.md (hunk 3 above)
Comment (jerdra) [reply README.md]: thanks for the catch :)

File: README.md [hunk 4]
```diff
@@ -1,9 +1,44 @@
...
+[![Slack Status](https://img.shields.io/badge/Slack_Channel-neuroimaging-E01563.svg)](https://swcarpentry.slack.com/messages/CCJBHKCHZ)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/carpentries-incubator/SDC-BIDS-fMRI/gh-pages)
 
+## Background
+
+This is one sub-module within the [Neuroimaging cirriculum][neuro_cirriculum]. Visit the link to view all the modules associated with the Neuroimaging Carpentries program.
+
+**fMRI Analysis in Python** is a programme developed to facilitate reproducible analysis in the area of functional neuroimaging analysis. Python is emerging as a standard language of data analysis, visualization, and workflow building. More recently, it has rapidly been adopted by the neuroimaging community as a means of developing powerful open-source tools in favour of historially used opaque packages such as AFNI, FSL and SPM. In addition, the barrier to entry to python is low - meaning that you as the user can easily develop your own packages and contribute to the open-source codebase of neuroimaging!
```
Comment (josephmje) [inline README.md:11]:
```suggestion
**fMRI Analysis in Python** is a programme developed to facilitate reproducibility in functional neuroimaging analyses. Python is emerging as a standard language of data analysis, visualization, and workflow building. More recently, it has rapidly been adopted by the neuroimaging community as a means of developing powerful open-source tools in favour of historically used opaque software such as AFNI, FSL and SPM. In addition, the barrier to entry to Python is low - meaning that you as the user can easily develop your own packages and contribute to the open-source codebase of neuroimaging!
```

File: README.md [hunk 5]
```diff
@@ -1,9 +1,44 @@
...
+
+***
+
+The **fMRI Analysis in Python** is a workshop series started up via a collaboration between researchers and staff at the Centre for Addiction and Mental Health (Toronto, ON), the University of Western Ontario (London, Ontario), and McGill University (Montreal, Quebec).
+
+***
+
+## About the lesson
+
+This lesson covers fMRI imaging analysis from the basic steps of preprocessing and data cleaning, to running an analysis to explore connectivity patterns in the brain.
```
Comment (josephmje) [inline README.md:22]:
```suggestion
This lesson covers fMRI imaging analysis from the basic steps of preprocessing and data cleaning, to running an analysis, to exploring connectivity patterns in the brain.
```

**Review by josephmje — COMMENTED** [review]
Looks good to me! Just made a few suggestions to the Background

**Pull request merged**